src/main/java/com/denizenscript/clientizen/mixin/EntityRenderedMixin.java

```diff
@@ -0,0 +1,30 @@
+package com.denizenscript.clientizen.mixin;
+
+import com.denizenscript.clientizen.events.EntityStartsStopsRenderingScriptEvent;
+import com.denizenscript.clientizen.objects.EntityTag;
+import net.minecraft.client.render.Frustum;
+import net.minecraft.client.render.entity.EntityRenderDispatcher;
```

Comment:
Some of these imports seem to be unused? can `Ctrl + Alt + O` in Intellij to do an imports cleanup

Reply:
None of the imports are actually unused. Using `Ctrl + Alt + O` does nothing. All the things that look unrelated are used for the Mixin to work and inject the method.